### ISSUE TITLE
Does Data Capture work in non WGS coordinates?

### ISSUE THREAD

**Issue by HeatherHillers** [DMS-Aus/Roam#92]
I have been doing my custom save Thing, so I am only now running into this.

If I make take one of my Projects and create a Point layer and try to add to it using a form
created by config manager (simple, simple), the Point gets added but I see nothing.  

And I am realising that this is because my Projects are Gauß Krüger projected.

If I take a Project built in WGS84 that works great (like my simple Test Project, built in the config Manager, with just a empty Point layer with one id Attribute, and the layer is empty) and I Change the qgis Project so that the project crs and the layer crs is Gauß Kruger, I can't capture.

Do I have to have everything in WGS 84 Pseudo Mercator for Roam to work?

**Comment by PauloEduardoCardoso** [comment]
I've noticed that ROAM will not accept data entry when I use a modified Albers projection (user projection) nor with UTM 34S (South Africa). It will work just fine with WGS84.

I have all layers but the raster map in WGS84. Base raster layer is a modified Albers projection.

This modified Albers works just fine in QGIS and even in ROAM (everything is perfectly aligned).

Should one keep all layers (vector and raster) in the same projection?

Modified Albers projection:
`+proj=aea +lat_1=-18 +lat_2=-32 +lat_0=0 +lon_0=24 +x_0=0 +y_0=0 +datum=WGS84 +units=m +no_defs`

**Comment by rjwillson** [comment]
check to see if the CRS of the QGIS project is set to the same CRS as the data layers (containers) that you want to collect the data into. In my experience the rest of the supporting layers can have other CRSs and will be projected on the fly. However, if I forget to save the QGIS project with the CRS of the data layers because I changed it for other purposes, and then open Roam, I can only collect features that have missing spatial geometry and they will not show up on the map

**Comment by PauloEduardoCardoso** [comment]
True. That may be the case. My layers are all in WGS84 but the base map is Albers. Using the project with WGS84 works just fine.

Will ROAM support different projection in the future?

By now I'll keep all layers in a single projection.

Thanks!

**Comment by NathanW2** [comment]
I would consider it a bug for now as I have to handle the case in the map
tools.

**Comment by TonyFMCMC** [comment]
Using Roam 2.6 on Windows 8. in QGIS my project is EPSG 7855, and my layers are native in 7855.  The layers are MSSQL.  I can add a polygon in QGIS, and it saves in the right place.  In Roam the polygon saves data, but the location is way off the map.  Is there some confusion between metres as units and degrees?
Or do I need to convert my layers to WGS84, and do the whole thing in WGS84?
Cheers!

**Comment by rjwillson** [comment]
I think the mostly likely problem is that your layers are not truly in EPSG 7855. Best way to ensure layers are in a specific CRS is to do a save as and change to 7855. But what version of Q are you using? Using QGIS 3 causes some problems with the CRS.

**Comment by TonyFMCMC** [comment]
Ah, I am using Q3.  Will try with 2.8 and let you know.  Thanks rjwillson!

**Comment by TonyFMCMC** [comment]
I changed to Q2.18, and opening the project showed that the project properties was in WGS84. changing this to be consistent with the layers in 7855 then fixed the problem.  Thanks for your help!
